packages/picasso.js/src/core/component/component-factory.js

```diff
@@ -431,8 +447,23 @@ function componentFactory(definition, context = {}) {
   let currentNodes;
   let preComputedRect;
 
+  function updateBrushNodes(nodes) {
+    const { rendererSettings } = settings;
+    const progressive = typeof rendererSettings?.progressive === 'function' && rendererSettings.progressive();
+    if (progressive) {
+      if (progressive.start === 0) {
```

Comment:
why do brusing have this reset when progressive.start === 0?
there is no matching logic for the data part in component-factory.js

Reply:
We need to store previously-rendered-nodes in brush during progressive-rendering.

Reply:
I got your point! Thanks for that. Just updated!